src/components/v5/common/ActionSidebar/partials/Motions/steps/StakingStep/partials/StakingForm/helpers.ts

```diff
@@ -76,3 +76,24 @@ export const getPredictedPercentage = (
 
   return predictedPercentage;
 };
+
+/**
+ * Function returning the max amount user can stake which is the smaller of
+ * the activated tokens balance and the remaining oppose or support stake amounts
+ */
+export const getMaxStakeAmount = (
+  voteType: MotionVote,
+  activatedTokens: BigNumber,
+  remainingStakes: string[],
```

Comment:
I don't have a huge issue with this, but maybe we could pass in remaining stakes in 2 parts, oppose and support remaining so it makes a bit more sense when reading it

Reply:
I thought about it (more specifically fixing it at the root, so in the schema) but GraphQL doesn't seem to support tuples. Since it's kind of an established convention in the codebase already, I'll leave it as such